src/miner.cpp

```diff
@@ -140,12 +140,15 @@ std::unique_ptr<CBlockTemplate> BlockAssembler::CreateNewBlock(const CScript& sc
     fIncludeWitness = IsWitnessEnabled(pindexPrev, chainparams.GetConsensus());
 
     fIncludeMWEB = IsMimblewimbleEnabled(pindexPrev, chainparams.GetConsensus());
+    if (fIncludeMWEB) {
```

Comment:
can we write this:
```
if (fIncludeMWEB) {
       mweb_builder = libmw::miner::NewBuilder(nHeight, pcoinsTip->GetMWView());
       AddHogExTransaction(pindexPrev);
}
```

Reply:
No. AddHogExTransaction has to be done after addPackageTxs. And mweb_builder must be created before calling addPackageTxs.

Reply:
I get it now